src/SFA.DAS.Assessor.Functions/Functions/Ofqual/OfqualDownloader.cs

```diff
@@ -0,0 +1,60 @@
+﻿using System;
+using System.Net.Http;
+using System.Threading.Tasks;
+using Microsoft.Extensions.Logging;
+using SFA.DAS.Assessor.Functions.Domain.Entities.Ofqual;
+using SFA.DAS.Assessor.Functions.Domain.OfqualImport.Interfaces;
+using SFA.DAS.Assessor.Functions.Infrastructure;
+
+namespace SFA.DAS.Assessor.Functions.Functions.Ofqual
+{
+    internal abstract class OfqualDownloader
+    {
+        private readonly IOfqualDownloadsBlobFileTransferClient _blobFileTransferClient;
+        private readonly HttpClient _httpClient;
+        private readonly OfqualDataType _ofqualFileType;
+
+        protected OfqualDownloader(
+            IOfqualDownloadsBlobFileTransferClient blobFileTransferClient,
+            HttpClient httpClient,
+            OfqualDataType ofqualFileType)
+        {
+            _blobFileTransferClient = blobFileTransferClient;
+            _httpClient = httpClient;
+            _ofqualFileType = ofqualFileType;
+        }
+
+        protected async Task<string> DownloadData(ILogger logger)
+        {
+            logger.LogInformation($"Downloading {_ofqualFileType} data from {_httpClient.BaseAddress}.");
+
+            try
+            {
+                using HttpResponseMessage response = await _httpClient.GetAsync("");
```

Comment:
string.Empty is recommended instead of "" as it generally indicates a deliberate absence of a string

Reply:
Changed